src/components/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -144,9 +145,22 @@ export default function ConfigPanel({
     <div className="config-panel">
       <div className="row">
         {isLoading ? (
-          <div className="is-loading u-vertically-center">
+          <div className="full-size u-vertically-center">
             <Spinner />
           </div>
+        ) : !isLoading && (!config || Object.keys(config).length === 0) ? (
+          <div className="full-size u-vertically-center">
+            <div className="config-panel__message">
+              <img
+                src={boxImage}
+                alt="3d box"
```

Comment:
Given this image is purely decorative and adds no semantic value, this alt should be blank.

Reply:
see above

Reply:
Please set an empty string.

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/alt-text.md#img